example/rds-cluster-tf/variables.tf

```diff
@@ -78,3 +78,7 @@ variable "subnets_tag_name" {
   default = "Private application *"
 }
 
```

Comment:
The configuration will pass to tf-modules/cumulus-rds-tf.  See example/rds-cluster-tf/main.tf.
You might need to test it separately since we usually don't build rds cluster.